tasks/install-helm.yml

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
Installing Helm on my system should not be the concern of this role.
I might be using a Docker image with a very specific Helm version to apply against this role.

Reply:
Helm comes as binary. What would be the benefit of using Docker?

Reply:
As discussed I will sadly remove the install logic :-(